### PR TITLE
Modifying the sermons admin view to put the filters on top.

### PR DESCRIPTION
**Pull request by vistamedia**
I modified the sermons view only. This should be enough to give you the right method to impement that properly in the JCB compiler.
Please give me a call when you have time ;) Have a nice day my friend.

### CODE REVIEW

**Comment by Llewellynvdm** [comment]
Thank you, this will help greatly to setup the new filter option for vdm-io/Joomla-Component-Builder#378

File: admin/models/fields/preachers.php [hunk 1]
```diff
@@ -48,23 +48,25 @@ class JFormFieldPreachers extends JFormFieldList
...
+		$query->select($db->quoteName(array('a.id','a.name'),array('id','preacher_request_id_name')));
+		$query->from($db->quoteName('#__sermondistributor_preacher', 'a'));
+		$query->where($db->quoteName('a.published') . ' = 1');
+		$query->order('a.name ASC');
+		$db->setQuery((string)$query);
+		$items = $db->loadObjectList();
+		$options = array();
+		if ($items)
+		{
+			if ($this->multiple === false) {
```
Comment (vistamedia) [inline admin/models/fields/preachers.php:62]: This should be implemented on custom fields to be able to use them as filters. Without this empty option will make the filter always active.

Comment (Llewellynvdm) [reply admin/models/fields/preachers.php:62]: Yes I have added a switch to manage this:
image

With these notices:
> The single filter selection option allows the user to select just one value in this filter/field.

> The multi filter selection option allows the user to select more then one value in this filter/field. Please note that this option only works with the new filters that load above the admin list view. You can select the new filter option when/where you add the view to the component.

This option is now shown when adding a field to the admin view.

Comment (Llewellynvdm) [reply admin/models/fields/preachers.php:62]: Then adding the admin view to the component I added this field:
image

Which will by default be the new above filter option.

**Pull request merged**